app/routes/admin/groups/components/GroupMembers.js

```diff
@@ -42,6 +42,9 @@ export const GroupMembers = ({
   fetch,
 }: Props) => (
   <div className={styles.groupMembers}>
+    {showDescendants || (
+      <>Antall medlemmer: {groupsById[groupId.toString()].numberOfUsers}</>
```

Comment:
Makes it easy to see how many users are in a given group (only current group, not child groups..)

![Screenshot from 2020-10-16 14-53-17](https://user-images.githubusercontent.com/1467188/96260483-5e49d800-0fbf-11eb-8326-2e1155b8080a.png)